### PR TITLE
Doppler (Independent Publisher)

### PR DESCRIPTION
**Pull request by developer-farhan**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ✔️] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ✔️] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ✔️] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ✔️] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ✔️] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ✔️] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ✔️] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [✔️ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [✔️ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

# `4 Unique Operations`
Doppler-Retrieve
Doppler-Update
Doppler-Delete
Doppler-List

# `Test Operation Section`
After_validation_1
After_validation_2

# `Paconn Validation`
Paconn Validation

# `Solution Checker Validation`
Solution Checker Validation

### CODE REVIEW

**Comment by developer-farhan** [comment]
@microsoft-github-policy-service agree

File: independent-publisher-connectors/Doppler/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2184 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Doppler Farhan Latif",
+    "description": "Doppler\u2019s developer-first security platform empowers teams to seamlessly manage, orchestrate, and govern secrets at scale.",
+    "version": "1.0",
+    "contact": {
```
Comment (vmanoharas) [inline independent-publisher-connectors/Doppler/apiDefinition.swagger.json:7]: can you please specify url as mentioned in the template in the below link?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json

File: independent-publisher-connectors/Doppler/readme.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
+# Doppler Secrets
+Doppler Secrets is a robust and user-friendly secrets management platform designed for developers and teams. It empowers users to efficiently handle and secure sensitive information such as API keys, tokens, and configuration variables across diverse projects. With features like version control, environment-specific configurations, and seamless integrations, Doppler simplifies the management of secrets, ensuring a streamlined and secure development workflow. Whether for small-scale projects or large enterprise applications, Doppler offers a scalable and reliable solution for maintaining and accessing critical secrets across various environments. With this connector, you can easily create, read, update, and delete secrets in Doppler’s Secret enabling you to automate the management of your secrets,configs,projects,project roles,project members.
```
Comment (vmanoharas) [inline independent-publisher-connectors/Doppler/readme.md]: can you please refer the below readme.md link and update the required fields?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json

**Review by vmanoharas — COMMENTED** [review]
Hello @officialFlutterDeveloper, 

Could you please resolve review comments?

File: independent-publisher-connectors/Doppler/apiDefinition.swagger.json (hunk 1 above)
Comment (developer-farhan) [reply independent-publisher-connectors/Doppler/apiDefinition.swagger.json:7]: Added the url

File: independent-publisher-connectors/Doppler/readme.md (hunk 2 above)
Comment (developer-farhan) [reply independent-publisher-connectors/Doppler/readme.md]: Reduced the number of character current is 400

File: independent-publisher-connectors/Doppler/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+          "tooltip": "Provide your Service Token",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#5e17eb",
```
Comment (vmanoharas) [inline independent-publisher-connectors/Doppler/apiProperties.json]: Please update the iconBrandColor as per the template here?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json

**Review by vmanoharas — COMMENTED** [review]
Hello @officialFlutterDeveloper, 

Can you please update the iconBrandColor?

File: independent-publisher-connectors/Doppler/apiProperties.json (hunk 3 above)
Comment (developer-farhan) [reply independent-publisher-connectors/Doppler/apiProperties.json]: Changed Brand Color To 'da3b01' | Added 'stackOwner' field in apiProperties.json

**Comment by vmanoharas** [comment]
Hello @officialFlutterDeveloper, 

Can you please correct the path of files? it seems that there is an empty file added as "git"
image

Please refer this link for other Independent connectors - https://github.com/microsoft/PowerPlatformConnectors/pull/3209/files

**Comment by developer-farhan** [comment]
@vmanoharas 'git' file has been deleted :)

**Review by vmanoharas — APPROVED** [review]
Hello @officialFlutterDeveloper 
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by developer-farhan** [comment]
@vmanoharas it was my honor working with you and the team. This has motivated me to work harder and make more useful and development needed connectors.

**Comment by vmanoharas** [comment]
> @vmanoharas it was my honor working with you and the team. This has motivated me to work harder and make more useful and development needed connectors.

Thank you @officialFlutterDeveloper . Your dedication and motivation are inspiring, and I look forward to witnessing the continued impact of your hard work.

**Comment by developer-farhan** [comment]
@vmanoharas I am reaching out to request an update on the status of my connector deployment

**Comment by vmanoharas** [comment]
Hello @officialFlutterDeveloper, 

Thank you very much for reaching out!

Your connector is deployed to South East Asia region, it will be completed to all regions by next week.

**Comment by developer-farhan** [comment]
@vmanoharas Requesting update on the connector deployement as the label has not been updated & how long it will take for my connector to show up [here](https://learn.microsoft.com/en-us/connectors/connector-reference/). Thanks :)

**Comment by vmanoharas** [comment]
Hello Partner,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Pull request merged**